ChangeLog

```diff
@@ -2,6 +2,10 @@ mbed TLS ChangeLog (Sorted per branch, date)
 
 = mbed TLS x.x.x branch released xxxx-xx-xx
 
+API Changes
+   * Add a common error code for a feature that is not supported by the
```

Comment:
I think we can be specific and say `MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED` has been added,

Reply:
Yes, specific is better.